### ISSUE TITLE
[NGRM] - Option to generate and combine separate renderings of each layer into textures/images/framebuffers. 

### ISSUE THREAD

**Issue by nilscb** [equinor/webviz-subsurface-components#1376]
When viewed in 2D there should be the possibility to draw each layer in separate frame buffers/textures (using z-buffer depth test) and then combine the resulting images so that the last drawn layers will be shown on top. For example all of wells on top of a map.

This will replace the current functionality of turning of and on z-buffer depth test for each layer ("depthTest" true/false keyword) which is not optimal for true 3D layers like the grid layer.

Some possible useful discussion/links on how to implement: https://github.com/visgl/deck.gl/issues/3313

**Comment by hkfb** [comment]
@nilscb can we rephrase this as a use case and leave the suggested implementation as a detail (there might be other ways to achieve the same goal)?

Eg. "Option to render layers in ortographic view superimposed, honoring layer ordering instead of z values"
